.pre-commit-config.yaml

```
@@ -1,16 +1,14 @@
+files: "PACKAGE_NAME\/"
 repos:
-- repo: https://github.com/pycqa/isort
-  rev: 5.10.1
-  hooks:
-  - id: isort
-    language_version: python3
-- repo: https://github.com/pycqa/flake8
-  rev: 6.0.0
-  hooks:
-  - id: flake8
-    language_version: python3
-- repo: https://github.com/psf/black
-  rev: 22.10.0
-  hooks:
-  - id: black
-    language_version: python3
+  - repo: https://github.com/psf/black
+    rev: 22.12.0
+    hooks:
+      - id: black
+        language_version: python3
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.230"
```

```suggestion
    rev: "v0.0.251"
```

that is the latest as of today but they have a rapid release cycle